lms/static/sass/multicourse/_account.scss

```diff
@@ -534,7 +534,7 @@
       }
 
       &.button-oa2-google-oauth2:hover, &.button-oa2-google-oauth2:focus {
-        background-color: #dd4b39;
+        background-color: #D73924;
```

Comment:
Good idea - in that case, will also modify FB/Linkedin to use their respective variables. Or not appropriate for this ticket?